### PR TITLE
Reorder test

### PR DESCRIPTION
**Pull request by sofiaso**

### CODE REVIEW

File: TAF_EasyRest/TestFramework/Pages/Client/OrderHistoryPage.cs [hunk 1]
```diff
@@ -9,52 +10,65 @@ public class OrderHistoryPage : BasePage
...
-            _historyButton.Click();
+            driver.WaitUntilElementIsVisible(_historyButton, timeToWait)
+                .Click();
             FillOdersList();
             return this;
         }
 
+        public override void GoToUrl()
+        {
+            Console.WriteLine(baseUrl + _pageUrl);
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/Pages/Client/OrderHistoryPage.cs:68]: why do you log into Console if you have a report for that purpose?

**Review by dmatsev — COMMENTED** [review]
I left the comment.

File: TAF_EasyRest/TestFramework/Pages/Client/OrderHistoryPage.cs (hunk 1 above)
Comment (sofiaso) [reply TAF_EasyRest/TestFramework/Pages/Client/OrderHistoryPage.cs:68]: Funny moment, my bad. This method has to do driver.Navigate.... Thx for comment.

**Review by dmatsev — APPROVED** [review]

File: TAF_EasyRest/TestFramework/PageComponents/Client/OrderPageComponent.cs [hunk 2]
```diff
@@ -4,19 +4,20 @@ public class OrderPageComponent
     {
         private IWebDriver driver;
         private int index;
+        private string number => _orderField.FindElement(By.XPath($"//div[@class='MuiGrid-item-120 MuiGrid-grid-xs-1-148'][1]")).Text;
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/PageComponents/Client/OrderPageComponent.cs:7]: Probably you won't find this element having an id inside XPath...

File: TAF_EasyRest/TestFramework/Test/ClientTests.cs [hunk 3]
```diff
@@ -105,13 +134,13 @@ public void SubmitOrderTest()
...
         {
             UserLogout(email);
+
+            driver.Quit();
             
             DBCleanup.ChangeOrderStatusByNumber("Waiting for confirm", orderNumber);
             DBCleanup.DeleteLastOrder();
-
-            driver.Quit();
+            DBCleanup.DeleteLastOrder();
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Test/ClientTests.cs:142]: Why do you do this method two times?

**Review by LoveMyDaughter — APPROVED** [review]
Please take a look at my questions.

**Pull request merged**